.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -0,0 +1,141 @@
+---
+name: New Release
+about: Propose a new release
+title: Release v0.x.0
+labels: ''
+assignees: ''
+
+---
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Release Process](#release-process)
+- [Announce the Release](#announce-the-release)
+- [Final Steps](#final-steps)
+
+## Introduction
+
+This document defines the process for releasing llm-d-inference-scheduler.
+
+## Prerequisites
+
+1. Permissions to push to the llm-d-inference-scheduler repository.
+
+2. Set the required environment variables based on the expected release number:
+
+   ```shell
+   export MAJOR=0
+   export MINOR=1
+   export PATCH=0
+   export REMOTE=origin
+   ```
+
+3. If creating a release candidate, set the release candidate number.
+
+   ```shell
+   export RC=1
+   ```
+
+## Release Process
+
+1. If needed, clone the llm-d-inference-scheduler [repo][repo].
```

Comment:
nit: I think you can use the implicit reference version (just `[repo]`) when the two match